localization/english/wc_mercenaries_l_english.yml

```diff
@@ -8,4 +134,705 @@
+ mercenary_company_vanguard_article:0 "the "
+ mercenary_company_vanguard:0 "Vanguard of [TITLE.GetNameNoTierNoTooltip]"
+# Gilnean
+# mercenary_company_wolfs_and_roses_article:0 "the "
+ mercenary_company_wolfs_and_roses:0 "Wolf N' Roses"
+ mercenary_company_gentleguard_article:0 "the "
+ mercenary_company_gentleguard:0 "Gentleguard of [TITLE.GetNameNoTierNoTooltip]"
+# Arathorian
+ mercenary_company_arathorian_blades_article:0 "the "
+ mercenary_company_arathorian_blades:0 "Arathorian Blades of [TITLE.GetNameNoTierNoTooltip]"
+# mercenary_company_trolls_demise_article:0 "the "
+ mercenary_company_trolls_demise:0 "Troll's Demise"
+# mercenary_company_sword_blood_and_honor_article:0 "the "
+ mercenary_company_sword_blood_and_honor:0 "Sword, Blood and Honor"
+# Hillsbrad
+ mercenary_company_stubborn_soldiers_article:0 "the "
+ mercenary_company_stubborn_soldiers:0 "Stubborn Soldiers of [TITLE.GetNameNoTierNoTooltip]"
+# mercenary_company_ashes_of_stormwind_article:0 "the "
+ mercenary_company_ashes_of_stormwind:0 "Ashes of Stormwind"
+# Lordaeronian
+ mercenary_company_hired_blades_article:0 "the "
+ mercenary_company_hired_blades:0 "Hired Blades of [TITLE.GetNameNoTierNoTooltip]"
+# mercenary_company_blades_rifles_and_scepters_article:0 "the "
+ mercenary_company_blades_rifles_and_scepters:0 "Blades, Rifles and Scepters"
+ mercenary_company_knights_of_lordain_article:0 "the "
+ mercenary_company_knights_of_lordain:0 "Knights of Lordain"
+# Dalaranian
+ mercenary_company_warmages_article:0 "the "
+ mercenary_company_warmages:0 "Warmages of [TITLE.GetNameNoTierNoTooltip]"
+# mercenary_company_fire_ice_and_arcane_article:0 "the "
+ mercenary_company_fire_ice_and_arcane:0 "Fire, Ice and Arcane"
+ mercenary_company_magic_experimenters_article:0 "the "
+ mercenary_company_magic_experimenters:0 "Magic Experimenters of [TITLE.GetNameNoTierNoTooltip]"
+# Alteraci
+ mercenary_company_savage_warriors_article:0 "the "
+ mercenary_company_savage_warriors:0 "Savage Warriors of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_trollslayers_article:0 "the "
+ mercenary_company_trollslayers:0 "Trollslayers of [TITLE.GetNameNoTierNoTooltip]"
+# Baradin
+ mercenary_company_fleeing_prisoners_article:0 "the "
+ mercenary_company_fleeing_prisoners:0 "Fleeing Prisoners of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_rogue_wardens_article:0 "the "
+ mercenary_company_rogue_wardens:0 "Rogue Wardens of [TITLE.GetNameNoTierNoTooltip]"
+# Tirassian
+ mercenary_company_proud_buccaneer_article:0 "the "
+ mercenary_company_proud_buccaneer:0 "Proud Buccaneer of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_sea_wizards_article:0 "the "
+ mercenary_company_sea_wizards:0 "Sea Wizards of [TITLE.GetNameNoTierNoTooltip]"
+ mercenary_company_glory_of_sea_article:0 "the "
+ mercenary_company_glory_of_sea:0 "Glory of the Sea"
+# mercenary_company_orcs_demise_article:0 "the "
+ mercenary_company_orcs_demise:0 "Orc's Demise"
```

Comment:
This name is problematic because:
- They may be no orcs (if we make a bookmark before the Dark Portal)
- They may be hired by orcs.